components/inputs/input-date.js

```diff
@@ -51,11 +54,13 @@ class InputDate extends LocalizeCoreElement(LitElement) {
 			/**
 			 * Value of the input
 			 */
+			validationCustomValid: { type: Boolean },
```

Comment:
Not sure if this needs to be public?

Reply:
Normally when using `d2l-validation-custom` the logic should be entirely contained within the event handler. You could do something along the lines of:
```
async _validate(e) {
    const isValid = !this._shownValue || isDateInRange(getDateFromISODate(this._shownValue), getDateFromISODate(this.minValue), getDateFromISODate(this.maxValue));
	e.detail.resolve(isValid);
}
```
`isValidationCustomValid` is public in the test component just so that I can assert that `validate()` returns a value that matches with the result returned from the `d2l-validation-custom`.